docs/cookbook/migrations/homebrew.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Homebrew
+description: Common questions and solutions for using Rust with Flox
+---
+
+# Homebrew migration guide
+Flox can replace Homebrew entirely, or they can be used together.
+
+This guide explains how to introduce Flox into environments where Homebrew is currently being used, either as a replacement or an addition. It introduces new concepts and proposes a basic procedure for mapping packages.
+
+## Why you might want to migrate
+Homebrew does a great job, and has been loved as the “missing package manager” for a generation of macOS users. But there are a few reasons you might consider moving to Flox:
+
+* **You want to install packages into discrete environments.** Flox allows you to create as many environments as you need, each with a different set of packages.
+* **You need a reproducible manifest.** Flox environments are defined in a TOML manifest that can be shared and reproduced - often across OS and system architectures.
+* **You need a broader set of packages.** Homebrew has a sizeable collection of packages, but the Flox Catalog is based on Nixpkgs, the largest collection of packages in the world.
+* **You need older versions of packages.** The Flox Catalog contains historical versions of each of its packages - in many cases, going back decades.
+
+## New concepts
+Flox works similarly to Homebrew, but there are a few new concepts to be aware of.
```

Comment:
"new concepts to be aware of" feels close to "be wary of" in this context. Could this section be flipped around to explain the happy path of the default env _first_ and then get into how all flox environments are virtual envs that need to be init and activated (but it was automatic and special for the default one)?

Reply:
OK - I restructured the doc and I feel a lot better about it now. Let me know if you like the new flow.

Reply:
Yes! Love it! That takes care of my concerns, and it flows much better overall imo.